pkg/controllers/customeradmin/rolebindings/rolebindings.go

```diff
@@ -0,0 +1,97 @@
+package common
+
+import (
+	rbacv1 "k8s.io/api/rbac/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+var (
+	// RoleBindingNames holds the customer-admin desired rolebindings
+	RoleBindingNames = map[string]rbacv1.RoleBinding{
+		"customer-admin": {
+			ObjectMeta: metav1.ObjectMeta{
+				Name: "customer-admin",
```

Comment:
still need to name these in a way that is unlikely to collide

Reply:
updated to `owned-by-osa-`